[OSDOCS#16946]: Make changes as per the errors and warnings for CQA-2 (#114626)
openshift/openshift-docs: modules/storage-azure-create-storage-class.adoc

[id="storage-create-azure-storage-class_{context}"]
= Creating the Azure storage class

[role="_abstract"]

> Storage classes are used to differentiate and delineate storage levels and usages. By defining a storage class, users can obtain dynamically provisioned persistent volumes.

Reviewer: ```suggestion
You can use storage classes to differentiate and delineate storage levels and usages. By defining a storage class, you can obtain dynamically provisioned persistent volumes.
```
